video/cloud-client/analyze/beta_snippets.py

```diff
@@ -0,0 +1,207 @@
+
+Usage Examples:
+    python beta_snippets.py boxes \
+    gs://python-docs-samples-tests/video/googlework_short.mp4
+
+    python beta_snippets.py \
+    emotions gs://python-docs-samples-tests/video/googlework_short.mp4
+
+    python beta_snippets.py \
+    transcription gs://python-docs-samples-tests/video/googlework_short.mp4
+"""
+
+import argparse
+
+from google.cloud import videointelligence_v1p1beta1 as videointelligence
+
+
+# [START video_face_bounding_boxes]
+def face_bounding_boxes(path):
+    """ Detects faces' bounding boxes. """
+    video_client = videointelligence.VideoIntelligenceServiceClient()
+    features = [videointelligence.enums.Feature.FACE_DETECTION]
+
+    config = videointelligence.types.FaceConfig(
+        include_bounding_boxes=True)
+    context = videointelligence.types.VideoContext(
+        face_detection_config=config)
+
+    operation = video_client.annotate_video(
+        path, features=features, video_context=context)
+    print('\nProcessing video for face annotations:')
+
+    result = operation.result(timeout=900)
+    print('\nFinished processing.')
+
+    # There is only one result because a single video was processed.
+    faces = result.annotation_results[0].face_detection_annotations
+    for i, face in enumerate(faces):
+        print('Face {}'.format(i))
+
+        # Each face_detection_annotation has only one segment.
+        segment = face.segments[0]
+        start_time = (segment.segment.start_time_offset.seconds +
+                      segment.segment.start_time_offset.nanos / 1e9)
+        end_time = (segment.segment.end_time_offset.seconds +
+                    segment.segment.end_time_offset.nanos / 1e9)
+        positions = '{}s to {}s'.format(start_time, end_time)
+        print('\tSegment: {}\n'.format(positions))
+
+        # There are typically many frames for each face,
+        # here we print information on only the first frame.
+        frame = face.frames[0]
+
+        time_offset = (frame.time_offset.seconds +
+                       frame.time_offset.nanos / 1e9)
+        box = frame.attributes[0].normalized_bounding_box
+
+        print('First frame time offset: {}s\n'.format(time_offset))
+
+        print('First frame normalized bounding box:')
+        print('\tleft  : {}'.format(box.left))
+        print('\ttop   : {}'.format(box.top))
+        print('\tright : {}'.format(box.right))
+        print('\tbottom: {}'.format(box.bottom))
+        print('\n')
+# [END video_face_bounding_boxes]
+
+
+# [START video_face_emotions]
+def face_emotions(path):
+    """ Analyze faces' emotions over frames. """
+    video_client = videointelligence.VideoIntelligenceServiceClient()
+    features = [videointelligence.enums.Feature.FACE_DETECTION]
+
+    # include_bounding_boxes must be set to True for include_emotions
+    # to work.
+    config = videointelligence.types.FaceConfig(
+        include_bounding_boxes=True,
```

Comment:
Is this still required? I thought we had them pull that?

Reply:
done.